clash-prelude/tests/Clash/Tests/Signal.hs

```diff
@@ -243,3 +249,58 @@ case_dynamicHasEffect = do
   assertBool "clk11+clk77 /= dclk11+dclk77lying" $
        (sampleMagicN (E.unsafeSynchronizer clk11 clk77 counter))
     /= (sampleMagicN (E.unsafeSynchronizer dclk11 dclk77lying counter))
+
+-- | Regression test
+case_changingDynamicClocks :: Assertion
+case_changingDynamicClocks = do
+  let
+    dclk11 = dynamicClockGen @H11 (fromList (cycle [10, 20, 30, 40, 50, 60, 70, 80, 90]))
+    dclk77 = dynamicClockGen @H77 (fromList (cycle [90, 80, 70, 60, 50, 40, 30, 20, 10]))
+
+    counter = fromList [(0 :: Int)..]
+    actual = P.take 70 (sample (E.unsafeSynchronizer dclk11 dclk77 counter))
+
+  print actual
```

Comment:
This seems to be a left over debugging statement which outputs text every test bench run